apps/reference/_api/usage.mdx

```diff
@@ -511,6 +514,18 @@ GET https://api.supabase.com/v1/projects/{ref}/functions
 ```
 
 
+<!-- AUTOGENERATED: DO NOT EDIT DIRECTLY IF THIS IS VERSION "next" -->
+
+</TabItem>
+
+<TabItem value="403">
```

Comment:
Do we return anything more specific for a 403 error? This tab is just empty and makes it look like the docs are broken:
![Screen Shot 2022-09-21 at 6 12 57 PM](https://user-images.githubusercontent.com/7026076/191637012-86d2ba6f-785f-42d1-ac2a-2f846ba8bac8.png)

Can we add something like we do with 500 errors?
![Screen Shot 2022-09-21 at 6 14 49 PM](https://user-images.githubusercontent.com/7026076/191637073-ce909ce3-4eab-4162-84e1-40098c24bdcc.png)

Reply:
I have no idea, this was autogenerated.

Reply:
Right. Any thoughts on how we can make this better? Should we just find a way to hide that tab if we have nothing to show there?

Reply:
Done, let's see what happens now.

Reply:
Oh was this not related to the PR at all? My bad!